src/components/graph/menus/root-network/root-network-node-editor.tsx

```diff
@@ -285,10 +289,7 @@ const RootNetworkNodeEditor: React.FC<RootNetworkNodeEditorProps> = ({
 
             updateRootNetwork(studyUuid, editedRootNetwork.rootNetworkUuid, rootNetworkInfos);
```

Comment:
not your scope, but updateRootNetwork promise error wont be caught here without .catch() clause.